docker/package/model.py

```diff
@@ -530,6 +530,69 @@ def __init__(
                     prestart_script="tezos-baking-prestart",
                 )
             )
+        custom_requires = []
+        for network in target_networks:
+            for proto in network_protos[network]:
+                custom_requires.append(f"tezos-baker-{proto.lower()}@custom@%i.service")
+                if proto not in self.noendorser_protos:
+                    custom_requires.append(
+                        f"tezos-endorser-{proto.lower()}@custom@%i.service"
+                    )
+        self.systemd_units.append(
+            SystemdUnit(
```

Comment:
This `SystemdUnit` is in a lot of ways copy-paste of the previous non-custom ones. It might be nice to separate it into some function to only pass arguments that differ to it... But maybe you don't think it's worth it, I'll leave it to you to decide.

Reply:
Yeah, they're quite similar indeed. The difference is `requires`, `description`, and `environment_file`. I agree that it's better to avoid copy-paste